server/src/internalClusterTest/java/org/opensearch/action/admin/cluster/repositories/RepositoryBlocksIT.java

```diff
@@ -55,39 +55,30 @@ public void testPutRepositoryWithBlocks() {
         logger.info("-->  registering a repository is blocked when the cluster is read only");
         try {
             setClusterReadOnly(true);
+            Settings.Builder settings = Settings.builder().put("location", randomRepoPath());
             assertBlocked(
-                client().admin()
-                    .cluster()
-                    .preparePutRepository("test-repo-blocks")
-                    .setType("fs")
-                    .setVerify(false)
-                    .setSettings(Settings.builder().put("location", randomRepoPath())),
+                OpenSearchIntegTestCase.putRepositoryRequestBuilder(
+                    client().admin().cluster(),
+                    "test-repo-blocks",
+                    "fs",
+                    false,
+                    settings,
+                    null
+                ),
                 Metadata.CLUSTER_READ_ONLY_BLOCK
             );
         } finally {
             setClusterReadOnly(false);
         }
 
         logger.info("-->  registering a repository is allowed when the cluster is not read only");
-        assertAcked(
-            client().admin()
-                .cluster()
-                .preparePutRepository("test-repo-blocks")
-                .setType("fs")
-                .setVerify(false)
-                .setSettings(Settings.builder().put("location", randomRepoPath()))
-        );
+        Settings.Builder settings = Settings.builder().put("location", randomRepoPath());
+        OpenSearchIntegTestCase.putRepository(client().admin().cluster(), "test-repo-blocks", "fs", false, settings);
     }
 
     public void testVerifyRepositoryWithBlocks() {
-        assertAcked(
-            client().admin()
-                .cluster()
-                .preparePutRepository("test-repo-blocks")
-                .setType("fs")
-                .setVerify(false)
-                .setSettings(Settings.builder().put("location", randomRepoPath()))
-        );
+        Settings.Builder settings = Settings.builder().put("location", randomRepoPath());
+        OpenSearchIntegTestCase.putRepository(client().admin().cluster(), "test-repo-blocks", "fs", false, settings);
```

Comment:
Today there are multiple places where we are calling the `ClusterAdminClient.preparePutRepository()` which is used for creating a repository in our cluster. We need all of our existing integration tests to work even with the `shard_path_type` repository metadata setting being `HASHED_PREFIX` OR `HASHED_INFIX`. To avoid the change being done at multiple places, I have tried making all the calls ultimately sink in at a central place where we can control the `shard_path_type`.